cloudformation-templates/zs_cc_cf_template_asg_gwlb.yaml

```diff
@@ -1,5 +1,5 @@
 AWSTemplateFormatVersion: 2010-09-09
-Description: Zscaler Cloud Connector Auto Scaling Group Template
+Description: Zscaler Cloud Connector Starter Deployment Template with Auto Scaling and GWLB v1.3.0
```

Comment:
Let's use the version in outputs, versioning in too many places will be a maintenance headache